tool/smarts/src/test/java/org/openscience/cdk/smarts/Mdl2SmartsTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * =====================================
+ *  Copyright (c) 2020 NextMove Software
+ * =====================================
```

Comment:
Can you plz add the license statement here too?

Reply:
Sorry default of the IDE - will fix,